app/code/community/Mailjet/Iframes/Helper/IframesWrapper.php

```diff
@@ -457,6 +456,7 @@ private function getToken()
      */
     private function generateToken()
     {
+        $this->setTokenAccess($this->tokenAccessAvailable);
```

Comment:
Please move this call outside of the `generateToken()` method, so it remains doing a single thing - generating token  - perhaps constructor is a good option. or even you can add a constructor argument where different values can be used for this on different places. You can still set these ones as a defaults